Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -2988,6 +2993,9 @@ - (void)yapDatabaseModified:(NSNotification *)notification
             if (scrollToBottom) {
                 [self scrollToBottomAnimated:shouldAnimateScrollToBottom && shouldAnimateUpdates];
             }
+            if (hasDeletions) {
+                [self cleanUpUnreadIndicatorIfNecessary];
```

Comment:
`cleanUpUnreadIndicatorIfNecessary` can be a little expensive, so only call it when we learn of deletions.